MetaMask/metamask-extension
# feat: accounts API feedback

## **Description**

This PR adds a link to the Snap Account feature feedback form

## **Manual testing steps**

1. Enable the Accounts API feature
2. Go to the accounts list > Click "Add account or hardware wallet"
3. Select "Add snap account"
4. In the "Create a Snap account" notice the Snap Account feedback link

## **Screenshots/Recordings**

![Screenshot 2023-10-10 at 12 23 57](https://github.com/MetaMask/metamask-extension/assets/17601467/76c46c97-ee6f-41e5-aef1-18518441446e)

## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/87

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
